docs/testing.md

```diff
@@ -0,0 +1,69 @@
+## Testing APB Image
+
+### Motivation
+As the ecosystem of APBs grows we want to facilitate a means for performing a basic sanity check to ensure that an APB is working as the author intended. The basic concept is to package an integration test with the APB code which will contain all of the needed parameters for the actions that the test playbook will run. 
+*Note: we are focusing on a basic provision to start other actions should be added in the future.*
+
+### Design
+The base APB entry point will learn the test action. The test action will be a user defined playbook. 
```

Comment:
I suggest creating a parallel directory for proposals here as we did in ansible-service-broker repo. 
```
docs/proposals
```